docs/limitations.md

```diff
@@ -26,6 +26,7 @@ Fortunately, as in theory records don't carry any relations information, usually
 
 > *If you don't emulate relations in your data schemas, you're safe.*
```

Comment:
So… if one does so, you can end up in an inconsistent state.

This is the case with something like Sync's bookmark model. There are two relations in every record: a parent pointer and a child list.

What changes need to occur in Kinto for this limitation to be lifted? This seems like the right doc to make a note of those.